.eslintrc.json

```diff
@@ -9,6 +9,6 @@
         "es6": true
     },
     "parserOptions": {
-        "sourceType": "module"
+        "ecmaVersion": 8
```

Comment:
This is what was causing your error: https://stackoverflow.com/a/53883984

I accidentally left the 'module' part in there.